### ISSUE TITLE
Documentation clarity

### ISSUE THREAD

**Issue by zenonp** [openstreetmap/mod_tile#387]
render_list -h says 
```
If you are using --all, you can restrict the tile range by adding these options:
  -x, --min-x=X        minimum X tile coordinate
```
etc. The word "coordinate", together with general knowledge and habit of how these things work, leads to the interpretation of "tile coordinate" as "bbox", which is wrong. This thing wants tile numbers, not tile coordinates. And you only realise this when you try a negative geographical coordinate after having rendered millions of useless tiles with "-x 3.47 -y 53.77" and other such, because render_list doesn't throw an error on decimal input either, although it expects integers for -x -X -y -Y. 

A better term for "tile coordinate" would be "tile number". But even so, a couple of lines of explanatory text could save thousands of collective hours of unnecessary work and subsequent frustration. 

Yet better would be if render_list actually did accept bbox geographical coordinates.

**Comment by hummeltech** [comment]
Hello @zenonp, yes, a bit of clarification could be helpful here, I have created a pull request to try and make things more clear. Let me know if that works for you.

Feel free to provide any suggestions to the pull request or open one yourself.

Thank you

**Comment by hummeltech** [comment]
Hello @zenonp, if you are compiling `mod_tile` yourself, you can now render using lat/lon bounds. This was merged in with #190 and the some updates to it were implemented in #417.
